scripts/globals/status.lua

```diff
@@ -93,6 +93,21 @@ xi.status =
     SHUTDOWN        = 20,
 }
 
+-----------------------------------
+-- Treasure Hunter 0 drop frequency
+-- https://ffxiclopedia.fandom.com/wiki/Treasure_Hunter
+-----------------------------------
+xi.frequency =
+{
+    SUPER_COMMON = 240,
```

Comment:
Maybe VERY_COMMON instead of SUPER_COMMON so that it matches the other side of the scale, and chart on bgwiki
https://www.bg-wiki.com/ffxi/Treasure_Hunter

Reply:
+1 for matching termonology